views/js/runner/helpers/messages.js

```diff
@@ -31,17 +31,18 @@ define([
      * @param {String} message - custom message that will be appended to the unanswered stats count
      * @param {String} scope - scope to consider for calculating the stats
      * @param {Object} runner - testRunner instance
+     * @param {Boolean} sync - flag for sync the unanswered stats in exit message and the unanswered stats in the toolbox. Default false
      * @returns {String} Returns the message text
      */
-    function getExitMessage(message, scope, runner) {
+    function getExitMessage(message, scope, runner, sync = false) {
```

Comment:
I missed that, but could you remove the default value from the function declaration? The browsers we are supposed to support don't have the support of this feature (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters)